src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let count = 0;
+
+  const adder = (...args) => {
```

Comment:
The spread operator `...args` is unnecessary since the `adder` function is expected to be called with a single argument each time, except when it's called with no arguments to return the sum. You can simplify the parameter handling by using a single argument instead.